include/capstone/riscv_insn.gen.inc

```diff
@@ -0,0 +1,984 @@
+/*=======================================================================*/
+/*This code was generated by the tool riscv_disasm_from_sail (see
+ * https://github.com/moste00/riscv_disasm_from_sail)*/
+/*from the sail model of RISC-V (see https://github.com/riscv/sail-riscv) @
+ * version 0b9c639f19da48734cbf14b61f6ad200b7c70de9.*/
+/*DO NOT MODIFY THIS CODE MANUALLY.*/
+/* */
+/*SPDX-License-Identifier: BSD-3-Clause*/
+/*=======================================================================*/
+
+#ifndef __Riscv_insn_gen_inc__
```

Comment:
Please use capital letters for include guards

Reply:
Also above in the other files please.